impl.js

```diff
@@ -163,7 +163,7 @@ function RPCServiceImplementation(service, impl, transforms = []) {
           }
         });
 
-        applyContextDefaults(data);
+        applyContextDefaults(data, child.resolvedRequestType.fields.context.resolvedType.fullName);
```

Comment:
Can we pull this out as a constant further up. Also there may not be a `context` field (or in fact any of the interim objects may be undefined) so will have to handle that case (we'll get an exception being thrown here)

Reply:
Struggled with this one, lots of null if I try and do anything with `child.resolvedRequestType.fields.context.resolvedType.fullName` or variations of - not sure what's happening there. It doesn't seem to get anything until you start going into the if/else functions. Is it only accessible inside the `wrappedImpl` functions?